schema.graphql

```diff
@@ -1160,14 +1160,16 @@ type Account @entity {
   updatedBlock: Block!
 }
 
+# Identity will have null values for `primaryAccount`, `eventId`, `datetime` in cases where Identity is referenced without actually being created as in Authorization/Instruction
+# Once Identity is created, values will be updated for these attributes
 type Identity @entity {
   id: ID!
   did: String! @index(unique: true)
-  primaryAccount: String! @index(unique: false)
+  primaryAccount: String @index(unique: false)
```

Comment:
This will be really annoying for every user of Identity (`id.?primaryAccount`). I think this should remain `String!` since this is something the chain ensures.